ShipperOptions.json

```diff
@@ -1751,29 +1751,29 @@
       },
       {
         "discounted_display": "UPS® Ground Saver Less than 1lb",
-        "display": "UPS SurePost® Less than 1lb",
+        "display": "UPS Ground Saver® Less than 1lb",
         "value": "92",
         "insurance": {
           "discounted_insurance": {"max": 1000}
         }
       },
       {
         "discounted_display": "UPS® Ground Saver 1lb or Greater",
-        "display": "UPS SurePost® 1lb or Greater",
+        "display": "UPS Ground Saver® 1lb or Greater",
         "value": "93",
         "insurance": {
           "discounted_insurance": {"max": 1000}
         }
       },
       {
-        "display": "UPS SurePost® BPM",
+        "display": "UPS Ground Saver® BPM",
```

Comment:
was there any way to hide BPM and Media from the DAP users?

Reply:
or would that more simply be done in code?

Reply:
Not without further development work in pappy. That seemed like a nice to have request, but it requires more work than the original issue request. So I didn't do it.